packages/edit-post/src/store/selectors.js

```diff
@@ -609,10 +609,13 @@ export const getEditedPostTemplate = createRegistrySelector(
 		const defaultTemplateId = select( coreStore ).getDefaultTemplateId( {
 			slug: slugToCheck,
 		} );
-		return select( coreStore ).getEditedEntityRecord(
-			'postType',
-			'wp_template',
-			defaultTemplateId
-		);
+
+		return defaultTemplateId
+			? select( coreStore ).getEditedEntityRecord(
+					'postType',
+					'wp_template',
+					defaultTemplateId
+			  )
+			: null;
```

Comment:
I checked the return type of `getEntityRecord` because sometimes the return type is important. And apparently it's gone through some recent changes; previously returning `undefined` and now returning `false` (see https://github.com/WordPress/gutenberg/pull/60988). That being said, it seems `null` is fine here even if that's different from the previous two return values with how `getEditedPostTemplate` is used since it's only looking for falsy values, not anything specific.